### PR TITLE
Update docs guide

### PR DESCRIPTION
**Pull request by efenfauzi**
docs related issue #20

### CODE REVIEW

File: docs/guide/security-conventions.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+This section provide instructions to comply with the standard convention how to secure code in development.
+
+## For Code: 
+### 1. Validate input. 
+Validate input from all untrusted data sources. Proper input validation can eliminate the vast majority of software vulnerabilities. Be suspicious of most external data sources, including command line arguments, network interfaces, environmental variables, and user controlled files [Seacord 05].
```
Comment (anak10thn) [inline docs/guide/security-conventions.md:5]: [Seacord 05]. <- delete this part please

File: docs/guide/standards.md [hunk 2]
```diff
@@ -4,16 +4,20 @@
...
+Software engineers must use these conventions to make the code more secure also more readable by another engineer: 
 
-We use [black](https://github.com/psf/black) for code formatter, and
-[pre-commit](https://pre-commit.com/) for it's
-[automation](https://github.com/psf/black#version-control-integration). Read
-[Black editor integration](https://github.com/psf/black#editor-integration) for
-more information.
+1. For code style, use the standard such as: 
+	1. Python 
+		* PEP-0257 - python officially recommends to follow the high-level docstring conventions. This describes how the classes, functions, and codes should be documented and how this really helps for code readability.
```
Comment (anak10thn) [inline docs/guide/standards.md:11]: add link

File: docs/guide/security-conventions.md (hunk 1 above)
Comment (efenfauzi) [reply docs/guide/security-conventions.md]: done..

File: docs/guide/standards.md (hunk 2 above)
Comment (efenfauzi) [reply docs/guide/standards.md]: done

**Pull request merged**